### PR TITLE
Add release signing key for android demo app

### PR DESCRIPTION
**Pull request by AMITGOELNY**
### _Summary_
Update CODEOWNERS to get notified on automated releases

* Add release signing key for demo app

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1,3 @@
 * @coinbase/wallet-mobile-sdk
+android/walletsdk/build.gradle @bangtoven @vishnumad @AMITGOELNY
+ios/CoinbaseWalletSDK/Resources/CoinbaseWalletSDK+version.swift @bangtoven @vishnumad @AMITGOELNY
```
Comment (bangtoven) [inline CODEOWNERS]: thanks for updating this.
i think we don't need this change now as it's addressed on the pipeline
image

File: android/example/build.gradle [hunk 2]
```diff
@@ -6,6 +6,15 @@ plugins {
...
 
+    signingConfigs {
+        development {
+            storeFile file('debug.keystore')
+            storePassword 'android'
+            keyAlias 'androiddebugkey'
+            keyPassword 'android'
+        }
+    }
+
```
Comment (bangtoven) [inline android/example/build.gradle:17]: i have to lean on to @vishnumad 's review 🙏

File: CODEOWNERS [hunk 3]
```diff
@@ -1 +1,3 @@
 * @coinbase/wallet-mobile-sdk
```
Comment (vishnumad) [inline CODEOWNERS]: Is the "Unknown owner: make sure the team @coinbase/wallet-mobile-sdk exists and has write access to the repository" warning expected?

File: android/example/build.gradle (hunk 2 above)
Comment (vishnumad) [reply android/example/build.gradle:17]: This looks good to me

**Review by vishnumad — DISMISSED** [review]

File: CODEOWNERS (hunk 3 above)
Comment (bangtoven) [reply CODEOWNERS]: afaik, yes, as heimdall also reads this file to do something with the cb infra stuff.

**Review by vishnumad — DISMISSED** [review]

**Review by bangtoven — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for bangtoven @ 2022-09-19 23:43:30 UTC
User cannot review their own commit

**Review by vishnumad — APPROVED** [review]

**Pull request merged**